### ISSUE TITLE
Beacons are triggered as alerts, instead of notifications

### ISSUE THREAD

**Issue by SimonErich** [beaconinside/ios-sdk#17]
Hi,

we are trying to implement beaconinside in our react-native app and created a react-native package containing your proximityservice and bridging it to react.

The code works perfectly on Android, but we are having trouble with the iOS part.

We are using the BeaconService 2.11.3 pod and the following implementation file in our package:
```#import "RNBeaconInside.h"
#import <Foundation/Foundation.h>
#import <React/RCTLog.h>
#import "BeaconService/BIBeaconService.h"

@implementation RNBeaconInside
RCT_EXPORT_MODULE(RNBeaconInside);

RCT_EXPORT_METHOD(performFetchWithCompletion){
    dispatch_async(dispatch_get_main_queue(), ^{
        [BIBeaconService performFetchWithCompletionHandler:nil];
    });
}

RCT_EXPORT_METHOD(handleNotification:(NSDictionary*) userInfo){
    dispatch_async(dispatch_get_main_queue(), ^{
        [BIBeaconService handleNotificationUserInfo:userInfo];
    });
}

RCT_EXPORT_METHOD(initWithToken:(NSString *)token)
{
    dispatch_async(dispatch_get_main_queue(), ^{
        [BIBeaconService requestAuthorization];
        [self startObserving];
        [BIBeaconService initWithToken:token];
    });
}

RCT_EXPORT_METHOD(terminate){
    dispatch_async(dispatch_get_main_queue(), ^{
        [[NSNotificationCenter defaultCenter] removeObserver:self];
        [BIBeaconService terminate];
    });
}

- (NSArray<NSString *> *)supportedEvents {
    return @[@"BeaconServiceRegionEnter",
             @"BeaconServiceRegionExit",
             @"BeaconServiceRegionUpdate",
             @"GeofenceServiceRegionEnter",
             @"GeofenceServiceRegionExit"];
}

- (void)startObserving {
    for (NSString *event in [self supportedEvents]) {
        [[NSNotificationCenter defaultCenter] addObserver:self
                                                 selector:@selector(handleReceivedEvent:)
                                                     name:event
                                                   object:nil];
    }
}

- (void)handleReceivedEvent:(NSNotification *)notification {
    [self sendEventWithName:notification.name body:notification.userInfo];
}

@end
```

As noted above:
The beacons and geofences are both triggered correctly, but not in form of a notification (as it should be and as they are displayed in Android), but in form of in app alert popups.

Since the documentation of the sdk is quite small, it might just be that we are calling a wrong method.
Any help would really be appreciated to get this working.

**Comment by rabschi** [comment]
Hello, when the app is open it shows an alert within the app, otherwise outside the app it's a regular Notification. Do you see these notifications? You can also directly contact me via c.rabsch@beaconinside.com to discuss our React implementation, which is work-in-progress.

**Comment by SimonErich** [comment]
Hello, thank your for answering that quick. I've sent you an email, but we can also have the discussion here for future refrence or users with the same problem.

**Comment by rabschi** [comment]
Thanks for reporting, released in v2.12.1.
